### PR TITLE
update to go client v0.12.5

### PR DESCRIPTION
**Pull request by pmenglund**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -287,10 +287,8 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/posener/complete v1.1.1/go.mod h1:em0nMJCgc9GFtwrmVmEMR/ZL6WyhyjMBndrE9hABlRI=
 github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4/go.mod h1:xMI15A0UPsDsEKsMN9yxemIoYk6Tm2C1GtYGdfGttqA=
-github.com/rockset/rockset-go-client v0.12.3 h1:Iq9bwJiA0W97CNi9Po9Xky/0culPiVctwamjzyBGv1A=
-github.com/rockset/rockset-go-client v0.12.3/go.mod h1:Apc/bRmbRrpBvgwtXFH3L3Afyk1tDSBZATtv0Nmb2LI=
-github.com/rockset/rockset-go-client v0.12.4 h1:0YkjFk/EN9/Rx2Lz3UNg4q35Pv+BM6UEJMtBf34/ko4=
-github.com/rockset/rockset-go-client v0.12.4/go.mod h1:Apc/bRmbRrpBvgwtXFH3L3Afyk1tDSBZATtv0Nmb2LI=
```
Comment (benhannel) [inline go.sum:293]: Why did it depend on multiple version before?

**Comment by benhannel** [comment]
Should I be concerned about the failing circle ci test?

File: templates/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ The preferred configuration method is by environment variables:
 
 `ROCKSET_APIKEY` and `ROCKSET_APISERVER`
 
-At this point in time the only valid option for api server is `api.rs2.usw2.rockset.com`
+At this point in time the only valid option for api server is `https://api.rs2.usw2.rockset.com`
```
Comment (benhannel) [inline templates/index.md:28]: Should we remove this comment since use east is coming online?

**Review by benhannel — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (pmenglund) [reply go.sum:293]: lack of cleanup - `go mod tidy` fixed it

File: templates/index.md (hunk 2 above)
Comment (pmenglund) [reply templates/index.md:28]: is it publicly available now?

**Pull request merged**